pkgs/build-support/fetchurl/default.nix

```diff
@@ -124,6 +124,8 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
 
   inherit curlOpts showURLs mirrorsFile impureEnvVars postFetch downloadToTemp executable;
 
+  userAgent = "Nix ${builtins.nixVersion} on nixpkgs ${stdenv.lib.nixpkgsVersion}";
```

Comment:
Not ideal (hard to track back to what it is) but sounds good to me.

On Wed, Aug 31, 2016 at 10:17 PM Daniel Peebles <notifications@github.com>
wrote:

> In pkgs/build-support/fetchurl/default.nix
> <https://github.com/NixOS/nixpkgs/pull/17757#discussion_r77104796>:
>
> > @@ -124,6 +124,8 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
> >
> >    inherit curlOpts showURLs mirrorsFile impureEnvVars postFetch downloadToTemp executable;
> >
> > +  userAgent = "Nix ${builtins.nixVersion} on nixpkgs ${stdenv.lib.nixpkgsVersion}";
>
> Could be the hash of the fetchurl files, I suppose!
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/17757/files/75b22922b8f285cf0e36ab900a807e3ca5f4ca6f#r77104796>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAErrNsyIUwykglYvGFe5RECxzM05YBiks5qljXEgaJpZM4Jkbay>
> .
>